pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -161,7 +171,7 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 
 		if len(args) >= 2 {
 			newName = args[1]
-		} else {
+		} else if oldRc != nil {
```

Comment:
Wait, why would oldRc ever be nil here? If we didn't find it, wouldn't we would have returned already inside the block starting at 137?